Fixed collectionPath and added a warning note. (#1828)
mulesoft/docs-mule-runtime-3: modules/ROOT/pages/dataweave-streaming.adoc

</order>
----
​
Given this XML source data, you can set the unit of the stream to `<order-item/>` by setting `collectionPath=order.order-items` in the `outputMimeType` value:

[source,XML,linenums]
----
<flow name="dw-streaming-example" >
  <http:listener doc:name="Listener"
      outputMimeType="application/xml; collectionpath=order.order-items; streaming=true"
      config-ref="HTTP_Listener_config" path="/input"/>
</flow>
----

> Note that you need to set both `streaming=true` and the `collectionPath`, if either of those is missing it will not stream the content.

Reviewer: ```suggestion

Note that you need to set both `streaming=true` and the `collectionPath` value. If either  is missing, DataWeave will not stream the content.
```
